pkg/controller/account/account_controller.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/go-logr/logr"
 	"github.com/openshift/aws-account-operator/pkg/controller/utils"
 	"github.com/openshift/aws-account-operator/pkg/localmetrics"
+	"github.com/openshift/operator-custom-metrics/pkg/wrappers"
```

Comment:
Ultimately, this file's delta would need to be applied similarly to all the controllers.